src/jbi/__init__.py

```diff
@@ -0,0 +1,24 @@
+"""
+Module domain specific code related to JBI.
+
+This part of the code is not aware of the HTTP context it runs in.
+"""
+from enum import Enum
+from typing import Dict, Tuple
+
+
+class Operation(str, Enum):
+    """Enumeration of possible operations logged during WebHook execution."""
+
+    HANDLE = "handle"
+    EXECUTE = "execute"
+    IGNORE = "ignore"
+
+    CREATE = "create"
+    UPDATE = "update"
+    DELETE = "delete"
+    COMMENT = "comment"
+    LINK = "link"
```

Comment:
Is it expected/implied that a stakeholder would be limited to these Operations?  Or are these open to alternate results? (Is this too constraining or the right amount of limited choice?)

I like the use of these Enum's instead of one-off strs; in particular the logging of the operations really seems to make sense.

Reply:
I'm a bit fearful that "Operation" could be confusing/limiting for stakeholders--that is if there gets to be complex actions.